.github/workflows/ci-format.yml

```
@@ -57,17 +57,21 @@ jobs:
           python3.7 -m venv ${{env.venv_dir}}
           . ${{env.venv_dir}}/bin/activate
           pip install --upgrade pip wheel
-          pip install .[dev]
+          pip install -e .[dev,camera-obs,train,test] 
```

Is this `camera-obs,train,test` necessary ?
```suggestion
          pip install -e .[dev] 
```